client/landing/gutenboarding/constants.ts

```diff
@@ -8,6 +8,7 @@ import type { ValuesType } from 'utility-types';
  *
  */
 export const FLOW_ID = 'gutenboarding';
```

Comment:
Can we keep this one constant and assign it with a ternary here? I think that would save having to do the multiple ternary checks later.

Reply:
Oh - "Invalid hook call. Hooks can only be called inside of the body of a function component." I guess we could export `getFlowId` somewhere else, instead of a constant?